Dockerfile

```diff
@@ -33,14 +33,18 @@ RUN easy_install3 pip
 RUN easy_install pip
 RUN pip3 install -U virtualenv auxlib
 RUN pip2 install -U virtualenv auxlib
-RUN curl -O https://repo.continuum.io/miniconda/Miniconda2-4.2.11-Linux-x86_64.sh
-RUN bash Miniconda2-4.2.11-Linux-x86_64.sh -b -p /usr/local/miniconda/
-RUN ln -s /usr/local/miniconda/bin/conda /usr/local/bin/conda
 
 # UID and GID from readthedocs/user
 RUN groupadd --gid 205 docs
 RUN useradd -m --uid 1005 --gid 205 docs
 
 USER docs
 
+# Install miniconda as docs user
+WORKDIR /home/docs
+RUN curl -O https://repo.continuum.io/miniconda/Miniconda2-4.2.11-Linux-x86_64.sh
```

Comment:
@ericholscher pinned to the latest version. Anyway, as I said, it's pinned in the docker image at building time but it will be automatically updated by conda as it documentation says